doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -181,10 +207,9 @@ The following properties are used if the particle impact one of the boundaries o
 	For a particle to be accounted for in the fluid mesh, it has to overlap one or more vertices of this fluid mesh. If the initial mesh is too coarse in regards to the particle size, the particle may not be captured if it does not intersect the outer mesh walls. To avoid this, a box refinement can be added around the particle (See Box refinement documentation).
```

Comment:
```suggestion
	For a particle to be accounted for in the fluid mesh, it has to overlap at least one vertices of this fluid mesh. If the initial mesh is too coarse in regards to the particle size, the particle may not be captured if it does not intersect the outer mesh walls. To avoid this, a box refinement can be added around the particle (See Box refinement documentation).
```